.github/workflows/schedule-stale.yml

```diff
@@ -0,0 +1,35 @@
+on:
+  schedule:
+    - cron: "0 16 * * *"
+name: Stale Bot workflow
+jobs:
+  build:
+    name: stale
+    runs-on: ubuntu-latest
+    steps:
+      - name: stale
+        id: stale
+        uses: actions/stale@v3
+        with:
```

Comment:
I noticed Gatsby landed this with dry run/debug only mode first, is that something we'd want to follow suit with?

> You can run this action in debug only mode (no actions will be taken on your issues) by passing debug-only true as an argument to the action.
> https://github.com/actions/stale/tree/ade4f65ff5df7d690fad2b171eeb852f4809dc0b#debugging

Reply:
Ah yea that'd be a good idea, can do